subgraphs/marketplace/template.yaml

```diff
@@ -283,6 +303,27 @@ dataSources:
         - event: Transfer(indexed address,indexed address,indexed uint256)
           handler: handleTransfer
       file: ./src/mappings/smol-bodies.ts
+  - name: Smol Bodies Pets
+    kind: ethereum/contract
+    network: {{ network }}
+    source:
+      address: "{{ smol_bodies_pets_address }}"
+      abi: ERC721
+      startBlock: {{ smol_bodies_pets_start_block }}
+    mapping:
+      kind: ethereum/events
+      apiVersion: 0.0.6
+      language: wasm/assemblyscript
+      entities:
+        - Transfer
+        - SmolBodiesMint
```

Comment:
Just noticed this entity - will it cause any issues here?

Reply:
No, this field is unused by the graph currently. It just has to be filled out.